### DISCUSSION TITLE
How to fix new report system

### DISCUSSION THREAD

**Discussion by humanpride** [ValveSoftware/Dota2-Gameplay#14702] [Ideas]
I would like to invite the game developers to add the following player-friendly innovations:
1. A detailed report on complaints from other players about you (in what game and for what was the complaint, it is even better if the complaint verified by the Patrol will be available for viewing in the replay - as the Patrol itself sees it);
2. Guide to good behavior in the game. What behavior is considered bad and for which you can receive a complaint. What behavior is considered good - what will allow players to raise their decency;
3. Personal recommendations for players on how they can correct their behavior. For example, if a player receives a lot of complaints about insults, then inform the player that he often insults other players, with obstruction of the game - inform that he often receives complaints about obstruction of the game and that he needs to play more carefully and help his team.

**Comment by BlameBenny** [comment]
This REALLY needs an upvote, I so agree with this guy.
